person.rb

```diff
@@ -1,24 +1,28 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Math.random(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+    super()
   end
```

Comment:
- [ ] Please kindly also implement the feedback for the `Book` class in the `Person` class.